althea_types/src/interop.rs

```diff
@@ -123,13 +123,21 @@ pub struct ExitClientIdentity {
     pub reg_details: ExitRegistrationDetails,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
+pub enum ExitVerifMode {
+    Email,
+    Off,
+}
+
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
 pub struct ExitDetails {
     pub server_internal_ip: IpAddr,
     pub netmask: u8,
     pub wg_exit_port: u16,
     pub exit_price: u64,
     pub description: String,
+    #[serde(skip_serializing_if = "Option::is_none", default)]
+    pub verif_mode: Option<ExitVerifMode>,
```

Comment:
Do you need Option here if it will be always `Some`? What would `None` mean? Or maybe there is no need to have `ExitVerifMode::Off` if `verif_mode.is_none()` means the verification is off?

Reply:
none is the current state (since we don't say right now) for backwards compat